Common/src/interface_interpolation/CNearestNeighbor.cpp

```diff
@@ -169,10 +169,7 @@ void CNearestNeighbor::SetTransferCoeff(const CConfig* const* config) {
     }
     END_SU2_OMP_PARALLEL
 
-    delete[] Buffer_Send_Coord;
     delete[] Buffer_Send_GlobalPoint;
-
-    delete[] Buffer_Receive_Coord;
```

Comment:
The 1:1 replacement of `delete[] Buffer_Send_Coord` would probably be `Buffer_Send_Coord.resize(0,0)`. However this function seems to be called repeatedly in the `channel_2D` testcase, so let's keep the buffers.
In `CIsoparametric.cpp`, `CRadialBasisFunction.cpp` such `delete[]`s do also occur, and I also left them away although I don't know whether the buffers are used repeatedly.

Reply:
Or you assign an empty matrix to it, which is more STL style since STL containers usually don't "resize down"
```
Buffer_Send_Coord = su2activematrix();
```